flake.nix

```diff
@@ -53,5 +51,10 @@
           };
         })
       ];
+      perSystem = { self', ... }: {
+        packages.hps-main =
```

Comment:
Can we have a more descriptive name?

Reply:
Sure. Does `hps-production-flags` sound good? production isn't quite accurate but strict would be confusing.